src/main/java/com/guysfromusa/carsgame/v1/CarResource.java

```diff
@@ -84,4 +82,15 @@ public Long removeCar(@ApiParam(name = "name", value="Car name to be deleted") @
         return carService.deleteCarByName(name);
     }
 
+    @PostMapping(path= "{name}/fix")
```

Comment:
I think better to use PutMapping as finally this is an update of the car

Reply:
and I'm thinking about this /fix as the only think we can update is to make car not crashed

Reply:
there might be issue with spring test with PUT mapping